### ISSUE TITLE
The tab order on the Edit Profile page holds a few surprises across browsers

### ISSUE THREAD

**Issue by rpkoller** [AzuraCast/AzuraCast#6679]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

#d64b1a4 (2023-10-20 0:01)

### What happened?

The tab order on https://demo.azuracast.com/profile is not in line with WCAG 2.1 SC 2.4.3. I've tested on MacOS Ventura with the latest browser versions: 

On Safari and Microsoft Edge the tab order is: 
`Name -> -Email Address -> Time Display -> Close -> Save Changes` (Language is inaccessible)
The shift tab order is: 
`Save Changes -> Close -> Ukranian -> E-Mail Address -> Name`  (Time display is inaccessible) 

=> Safari and Edge behave correctly in regards getting to a radio button group and tab out of it and within the radio button group you navigate with the arrow keys but it makes in one direction the language group inaccessible while in the other way around the time display group. 

On Firefox the tab order is: 
```Name->Email-Address -> Prefer System Default -> 12 Hour -> 24 Hour -> Use Browser Default -> English (Default) -> .... (the rest of the language options) ... -> Close -> Save Changes``` 
With shift tab the other way around. 

=> Firefox handles all radio button groups correctly and doesn't hide one in each direction like Safari and Edge but making each radio button groups option tab able is also not the usual behavior unfortunately :/

**Comment by BusterNeece** [comment]
@rpkoller Figured this one out after some digging: turns out Chromium groups checkboxes/radios all together as one unless they're distinguished by a `name` attribute. While we programmatically don't have any use for that attribute, it obviously helps the accessibility tools group the inputs.

Should be fixed in the latest Rolling Release/demo.

**Comment by rpkoller** [comment]
Nice! just checked with all three browsers. can confirm that the skipping of one group in each direction is fixed and all three browsers behave correctly. also in regards of firefox letting me tab through all options for time display is fixed. it only remains for the language category. there might be an easy follow up that would also fix that for safari and edge if you tab from `time display` to `language` you get to `use browser default` while when you shift tab from the `close` button you get to the `ukranian` option. 
perhaps it would make sense to select a default option for `language` as well and pick the `use browser default` option for that. that way the differing back and forth behavior for safari and edge would be gone and for firefox you wouldnt be able to tab through the options anymore but use the cursor keys to change the radio button selection.
